bundles/org.openhab.binding.nibeuplink/src/main/resources/ESH-INF/thing/f1145-channel-groups.xml

```diff
@@ -6,32 +6,76 @@
 	<channel-group-type id="f1145-general">
 		<label>General Channels</label>
 		<channels>
-			<channel id="44302" typeId="f1145-type-44302" />
-			<channel id="44270" typeId="f1145-type-44270" />
-			<channel id="43103" typeId="f1145-type-43103" />
+			<channel id="44302" typeId="type-energy">
+				<label>Heat Meter - Cooling Cpr EP14</label>
+				<description>Accumulated energy production as calculated by the heat meter</description>
+			</channel>
+			<channel id="44270" typeId="type-temperature">
+				<label>Calculated Cooling Supply Temperature S1</label>
+				<description>Calculated supply temperature in cooling mode for the climate system</description>
+			</channel>
+			<channel id="43103" typeId="type-hpac-state">
+				<label>HPAC State</label>
+				<description>State of the HPAC accessory</description>
+			</channel>
 		</channels>
 	</channel-group-type>
 	<channel-group-type id="f1145-compressor">
 		<label>Compressor Channels</label>
 		<channels>
-			<channel id="43424" typeId="f1145-type-43424" />
-			<channel id="43420" typeId="f1145-type-43420" />
-			<channel id="43416" typeId="f1145-type-43416" />
-			<channel id="40022" typeId="f1145-type-40022" />
-			<channel id="40019" typeId="f1145-type-40019" />
-			<channel id="40018" typeId="f1145-type-40018" />
-			<channel id="40017" typeId="f1145-type-40017" />
-			<channel id="40015" typeId="f1145-type-40015" />
-			<channel id="40016" typeId="f1145-type-40016" />
-			<channel id="43439" typeId="f1145-type-43439" />
+			<channel id="43424" typeId="type-defrosting-state">
+				<label>Tot. HW Op.time Compr. EB100-EP14</label>
+				<description>Total compressor operation time in hot water mode</description>
+			</channel>
+			<channel id="43420" typeId="type-defrosting-state">
+				<label>Tot. Op.time Compr. EB100-EP14</label>
```

Comment:
It is the original name which is also displayed by the API. Of course some labels could be replaced by more readable ones but this could also lead to very long labels. I have also seen a discussion in other bindings that there were complaints because labels were too long.